db/migrate/20180820193511_create_goods_commitments.rb

```diff
@@ -0,0 +1,15 @@
+class CreateGoodsCommitments < ActiveRecord::Migration[5.2]
+  def change
+    create_table :goods_commitments do |t|
+      t.references :user, foreign_key: true
+      t.boolean :verified
+      t.float :latitude
```

Comment:
let us save the location directly to inventory table to allow independent movement of resources.